apps/api/src/app/db/notifications/notifications.service.ts

```diff
@@ -329,6 +334,10 @@ export class NotificationsService {
         // Put stale notifications back into the processing queue
         await this.cleanUpStaleNotifications();
 
+        const [sec, ns] = process.hrtime(tickStart);
+        const elapsedMs = (sec * NS_PER_SEC + ns) / NS_PER_MS;
+        this.logger.debug(`NotificationsService wakeup completed in ${elapsedMs}ms. Processed ${processed} notifications. Sleeping...`);
```

Comment:
This will be kind of noisy, but it's logging to the `debug` sink, so that's what we want.